CK2Plus_expanded/history/titles/e_roman_empire.txt

```diff
@@ -0,0 +1,237 @@
+	holder =  145276 # Lucius
+}
+169.3.1 = {
+	holder =  145274 # Marcus Aurelius
+}
+180.3.17 = {
+	holder =  145275 # Commodus
+}
+193.1.1 = {
+	holder =  145273 # Pertinax
+}
+193.3.29 = {
+	holder =  145272 # Julian
+}
+193.6.1 = {
+	holder =  145269 # Septimus
+}
+211.2.4 = {
+	holder =  145271 # Geta
+}
+211.12.19 = {
+	holder =  145270 # Caracalla
+}
+217.4.8 = {
+	holder =  145268 # Macrinus
+}
+218.6.8 = {
+	holder =  145267 # Heliogabalus
+}
+222.3.11 = {
+	holder =  145266 # Alexander
+}
+235.3.18 = {
+	holder =  145265 # Maximinus
+}
+238.3.22 = {
+	holder =  145259 # Gordian
+}
+238.4.1 = {
+	holder =  145260 # Gordian II
+}
+238.4.12 = {
+	holder =  145264 # Pupienus
+}
+238.6.1 = {
+	holder =  145263 # Balbinus
+}
+238.7.29 = {
+	holder =  145262 # Gordian III
+}
+244.2.11 = {
+	holder = 70526 # Philip
+	law = crown_authority_2
+}
+249.9.1 = {
+	holder = 145258 # Decius
+}
+251.6.1 = {
+	holder = 145257 # Trebonianus
+}
+253.7.1 = {
+	holder = 145256 # Aemilian
+}
+253.8.1 = {
+	holder = 145254 # Valerian
+}
+260.6.1 = {
+	holder = 145255 # Gallienus
+}
+268.8.1 = {
+	holder = 145252 # Claudius
+}
+270.1.15 = {
+	holder = 145253 #Quintillus
+}
+270.5.1 = {
+	holder = 145250 #Aurelian
+}
+275.7.1 = {
+	holder = 145249 #Severina
+}
+275.9.25 = {
+	holder = 145247 #Tacitus
+}
+276.6.1 = {
+	holder = 145248 #Florian
+}
+276.9.1 = {
+	holder = 145245 #Probus
+}
+282.10.1 = {
+	holder = 145242 # Carus
+}
+283.8.1 = {
+	holder = 145243 # Numerian
+}
+284.11.1 = {
+	holder = 145244 # Carinus
+}
+284.11.20 = {
+	holder = 145241 # Diocletian
+}
+285.7.21 = {
+	holder = 145293 # Maximian
+}
+305.5.1 = {
+	holder = 70524 # Constantius
+}
+306.7.25 = {
+	holder = 145240 # Valerius Severus
+}
+307.3.1 = {
+	holder = 145294 #Maxentius
+}
+308.11.11 = {
+	holder = 145238 # Licinius
+}
+310.4.1 = { # Ruled from the death of his father in 306 but was not recognized by the east until 310
+	holder = 70523 # Constantine The Great
+}
+337.5.22 = {
+	holder = 70520 # Constantine II
+}
+340.4.1 = {
+	holder = 70522 # Constans
+}
+350.2.27 = {
+	holder = 70521 # Constantius II
+}
+360.2.1 = {
+	holder = 145237 # Julian
+}
+363.6.26 = {
+	holder = 145235 # Jovian
+}
+364.2.26 = {
+	holder = 145231 # Valentinian I
+}
+375.11.17 = {
+	holder = 145233 # Gratian
+}
+375.11.22 = {
+	holder = 145234 # Valentinian II
+}
+392.5.15 = {
+	holder = 70534 # Theodosius I
+}
+395.1.23 = {
+	holder = 145227 # Honorius
+}
+423.7.2 = {
+	holder =  145229 # Valentinian III
+}
+455.3.17 = {
+	holder = 1000251000 # Petronius Maximus
+}
+455.5.31 = {
+	holder = 1000251001 # Avitus
+}
+457.4.1 = {
+	holder = 1000251002 # Majorian
+}
+461.11.19 = {
+	holder = 1000251003 # Libius Severus
+}
+467.4.12 = {
+	holder = 1000251004 # Anthemius
+}
+472.3.23 = {
+	holder = 1000251005 # Olybrius
+}
+472.11.2 = {
+	holder = 1000251006 # Glycerius
+}
+474.6.1 = {
+	holder = 1000251007 # Julius Nepos
+}
+475.10.31 = {
+	holder = 1000251008 # Romulus Augustulus
+}
+476.9.4 = {
+	holder = 0
```

Comment:
Iulius Nepos ruled in Dalmatia and was recognized by the East and Odoacer as the Western emperor until his death in 480.